libkbfs/md_journal_test.go

```diff
@@ -162,6 +170,36 @@ func checkIBRMDRange(t *testing.T, uid keybase1.UID,
 	}
 }
 
+func BenchmarkMDJournalBasicMDv2(b *testing.B) {
+	_, _, id, signer, ekg, bsplit, tempdir, j :=
+		setupMDJournalTest(b)
```

Comment:
Good point! Although I think Fred's branch makes `setupMDJournalTest` just accept a version, so I'm gonna wait until his is merged and adapt to that